pkg/generator/gcr/gcr.go

```diff
@@ -65,7 +65,7 @@ func (g *Generator) generate(
 	ts, err := tokenSource(ctx, esv1beta1.GCPSMAuth{
 		SecretRef:        (*esv1beta1.GCPSMAuthSecretRef)(res.Spec.Auth.SecretRef),
 		WorkloadIdentity: (*esv1beta1.GCPWorkloadIdentity)(res.Spec.Auth.WorkloadIdentity),
-	}, res.Spec.ProjectID, false, kube, namespace)
+	}, res.Spec.ProjectID, "", kube, namespace)
```

Comment:
Hum, not sure if I like an empty string better than a false. :D

Reply:
What I'm saying is that this doesn't signify anything when reading. I would have to open the function to figure out what this is. Can you please create a constant saying EmptyStoreKind or something like that? That would make more sense when setting/reading an unused parameter. WDYT?